packages/lib/test/src/artifacts/Contracts.test.js

```diff
@@ -0,0 +1,50 @@
+'use strict'
+require('../../setup')
+
+import Contracts from '../../../src/artifacts/Contracts'
+
+contract('Contracts', () => {
```

Comment:
We've been using regular functions over arrow functions in tests blocks. If there isn't a scope-related reason to move from one to another, let's try to keep consistency.
```suggestion
contract('Contracts', function() {
```